### PR TITLE
Milestone: NY times clone

### PR DESCRIPTION
**Pull request by shubhsk88**
## NY times clone 
This PR address the milestone we achieved

- We built the fixed navbar  section 
- Use the grid layout to built the main links of the articles 
- Added the side section 
- Added the footer section with logo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,40 @@
-# newyork-times-clone
\ No newline at end of file
+# New York Times Webpage Clone
+
+This is a copy of the New York Webpage.
+
```
Comment (majovanilla) [inline README.md]: - [ ]  Improve your Readme file. It should be descriptive of the project. You are using the correct template but now you need to fill it with information about your project. You can add details about what you did in this project, why you did it, how you did it...Check this [link](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) for more info.

File: README.md [hunk 2]
```diff
@@ -1 +1,40 @@
...
+ [Live Demo](https://rawcdn.githack.com/shubhsk88/newyork-times-clone/1d0fc557878baa9914b92ae7c4c9916a7b2d4393/index.html)
+
+
+## Authors
+
+👤 **Mauricio Fow**
+
+👤 **Shubham Singh**
+
```
Comment (majovanilla) [inline README.md]: - [ ] Add information about you! Include your LinkedIn, Twitter or other social media so people can contact you!

File: index.html [hunk 3]
```diff
@@ -0,0 +1,806 @@
...
+          <div class="main__details">
+            <div class="date">March 17, 2014</div>
+            <div class="social">
+              <div class="icons"><i class="fab fa-facebook-f"></i></div>
+              <div class="icons"><i class="fab fa-twitter"></i></div>
+              <div class="icons"><i class="fas fa-envelope"></i></div>
+              <div class="icons"><i class="fas fa-share"></i></div>
+              <div class="icons-new"><i class="far fa-bookmark"></i></div>
+              <div class="icons-new"><i class="far fa-comment-alt"></i></div>
+            </div>
```
Comment (majovanilla) [inline index.html]: - [ ] The icons should also be links. :smile:

File: index.html [hunk 4]
```diff
@@ -0,0 +1,806 @@
...
+              Five Adaptable Recipes, All From Your Pantry
+            </li>
+            <li>
+              <img
+                src="https://static01.nyt.com/images/2020/03/22/nyregion/22diary-illos-slide-NLYF/22diary-illos-slide-NLYF-square640.jpg?quality=75&auto=webp&disable=upscale&width=350"
+                alt=""
+              />
+              His Grandmother Asked About One Woman She Said She Had Liked.’
+            </li>
+          </ul>
```
Comment (majovanilla) [inline index.html]: - [ ] The articles' titles on the aside should also be links.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,806 @@
...
+          <div class="main__details">
+            <div class="date">
+              <button class="button">Read 615 Comments</button>
+            </div>
+            <div class="social">
+              <div class="icons"><i class="fab fa-facebook-f"></i></div>
+              <div class="icons"><i class="fab fa-twitter"></i></div>
+              <div class="icons"><i class="fas fa-envelope"></i></div>
+              <div class="icons"><i class="fas fa-share"></i></div>
+            </div>
```
Comment (majovanilla) [inline index.html]: - [ ] The elements in this div should be links, too.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,806 @@
...
+          </div>
+        </div>
+      </div>
+      <div class="related">
+        <div class="related__content">
+          <div class="grid-container">
+            <h4>More in Space and Astronomy</h4>
+            <div class="related__content-grid">
+              <div class="grid-item">
+                <img
```
Comment (majovanilla) [inline index.html]: - [ ] All the articles on this section should contain links on their titles.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,806 @@
...
+                </h5>
+                <div class="date">March 19</div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="related__popular">
+          <div class="popular__title">
+            <h4>Most Popular</h4>
+          </div>
```
Comment (majovanilla) [inline index.html:657]: - [ ] All the titles on this section should be links.

**Review by majovanilla — CHANGES_REQUESTED** [review]
Hey team, 👋

## You did an *amazing*🌟 ✨ job on this project! 👊🏻  💯
You have met almost every requirement and your page looks very similar to the original one!!
There are still some changes 👇 you need to implement to approve this request 🔍

### Required Changes: ✍️

- Improve your pull request messages. Check this [example](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/)
- Please implement the required changes on the commented code below :point_down: 

### Optional Changes:

If you want to make your project even better 🔥💥, I have a few suggestions you can implement.  

These are completely [Optional]: 🦔

- Your project would look sooo much better if you include the hover effects on the links of the page, like the original version.

## Please, make all REQUIRED changes and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack (@majo)
I would be happy to help! 👩‍💻 


Happy coding 🤓

--Maria Reyes

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon:

Hi **TEAM**, congratulations on this great project 👍 You guys implemented all the requested changes pointed out by the previous TSE 😄 Awesome!!! The project is approved, go ahead and merge to master and submit a project completion form.
Before we merge let us add ```our emails``` to the author section of our README file and also let us update our live link to reflect our current changes 🙂 

Good luck with your next projects 👌 
_Happy coding_ 👨‍💻 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**